### PR TITLE
[DISCO-2567] update docker compose and remove nginx

### PR DESCRIPTION
**Pull request by taddes**
## References

JIRA: [DISCO-2567](https://mozilla-hub.atlassian.net/browse/DISCO-2567)
GitHub: [#165 ](https://github.com/mozilla-services/consvc-shepherd/issues/165)

## Description
Docker setup does not require nginx and in some cases causes a failure to validate certificates, blocking setup. 

This updates shepherd to remove this dependency so the default dev setup uses Docker.

## PR Review Checklist

_Put an `x` in the boxes that apply_

- [x] This PR conforms to the [Contribution Guidelines](https://github.com/mozilla-services/consvc-shepherd/blob/main/CONTRIBUTING.md).
- [x] The PR title starts with the JIRA issue reference, format `[DISCO-####]`, and has the same title (if applicable).
- [ ] [Documentation](https://github.com/mozilla-services/consvc-shepherd/tree/main/docs) has been updated (if applicable).
- [ ] Functional and performance test coverage has been expanded and maintained (if applicable).

[DISCO-2567]: https://mozilla-hub.atlassian.net/browse/DISCO-2567?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by tiftran — APPROVED** [review]
🎉 🎉

File: pyproject.toml [hunk 1]
```diff
@@ -20,6 +20,7 @@ sentry-sdk = "^1.23.1"
 psycopg2-binary = "^2.9.5"
 jsonschema = "^4.17.3"
 markus = {extras = ["datadog"], version = "^4.2.0"}
+docker = "^6.1.3"
```
Comment (tiftran) [inline pyproject.toml:24]: curious to know why is this needed?

Comment (taddes) [reply pyproject.toml]: I was playing around a bit, didn't remove! Got rid of it now!
